lib/src/ruby/module.cc

```diff
@@ -113,7 +116,7 @@ extern "C" {
         set_level(log_level::warning);
 
         // Initialize ruby
-        auto ruby = facter::ruby::api::instance();
+        auto ruby = api::instance();
```

Comment:
I vaguely lean towards statically initializing `api::ruby_lib_location`, but I think static initialization of module.cc contents vs `api::ruby_lib_location` may be non-deterministic.